examples/vpc_peering_common_with_autoscale/example.tfvars

```diff
@@ -0,0 +1,339 @@
+    }
+  },
+  fw-spoke1-vpc = {
+    vpc_name                        = "fw-spoke1-vpc"
+    create_network                  = true
+    delete_default_routes_on_create = true
+    mtu                             = "1460"
+    routing_mode                    = "REGIONAL"
+    subnetworks = {
+      fw-spoke1-sub = {
+        name              = "fw-spoke1-sub"
+        create_subnetwork = true
+        ip_cidr_range     = "192.168.1.0/28"
+        region            = "us-east4"
+      }
+    }
+    firewall_rules = {
+      allow-spoke1-ingress = {
+        name             = "allow-spoke1-vpc"
+        source_ranges    = ["192.168.0.0/16", "35.235.240.0/20", "10.10.12.0/28"]
+        priority         = "1000"
+        allowed_protocol = "all"
+        allowed_ports    = []
+      }
+    }
+  },
+  fw-spoke2-vpc = {
+    vpc_name                        = "fw-spoke2-vpc"
+    create_network                  = true
+    delete_default_routes_on_create = true
+    mtu                             = "1460"
+    routing_mode                    = "REGIONAL"
+    subnetworks = {
+      fw-spoke2-sub = {
+        name              = "fw-spoke2-sub"
+        create_subnetwork = true
+        ip_cidr_range     = "192.168.2.0/28"
+        region            = "us-east4"
+      }
+    }
+    firewall_rules = {
+      allow-spoke2-ingress = {
+        name             = "allow-spoke2-vpc"
+        source_ranges    = ["192.168.0.0/16", "35.235.240.0/20", "10.10.12.0/28"]
+        priority         = "1000"
+        allowed_protocol = "all"
+        allowed_ports    = []
+      }
+    }
+  }
+}
+
+# VPC Peerings
+
+vpc_peerings = {
+  trust-to-spoke1 = {
+    local_network_key = "fw-trust-vpc"
+    peer_network_key  = "fw-spoke1-vpc"
+
+    local_export_custom_routes                = true
+    local_import_custom_routes                = true
+    local_export_subnet_routes_with_public_ip = true
+    local_import_subnet_routes_with_public_ip = true
+
+    peer_export_custom_routes                = true
+    peer_import_custom_routes                = true
+    peer_export_subnet_routes_with_public_ip = true
+    peer_import_subnet_routes_with_public_ip = true
+  },
+  trust-to-spoke2 = {
+    local_network_key = "fw-trust-vpc"
+    peer_network_key  = "fw-spoke2-vpc"
+
+    local_export_custom_routes                = true
+    local_import_custom_routes                = true
+    local_export_subnet_routes_with_public_ip = true
+    local_import_subnet_routes_with_public_ip = true
+
+    peer_export_custom_routes                = true
+    peer_import_custom_routes                = true
+    peer_export_subnet_routes_with_public_ip = true
+    peer_import_subnet_routes_with_public_ip = true
+  }
+}
+
+# Static routes
+routes = {
+  fw-default-trust = {
+    name              = "fw-default-trust"
+    destination_range = "0.0.0.0/0"
+    vpc_network_key   = "fw-trust-vpc"
+    lb_internal_key   = "internal-lb"
+  }
+}
+# Autoscale
+autoscale_regional_mig = true
```

Comment:
Actually this parameters was left intentionally there - because otherwise this section of the code would have become a lot more complicated due to the fact that we need to iterate over `autoscale` variable within `lb_internal` module :

```
  backend_instance_groups = var.autoscale_regional_mig ? { for v in each.value.backends : v => module.autoscale[v].regional_instance_group_id } : merge([
    for v in each.value.backends :
    {
      for z_k, z_v in var.autoscale[v].zones :
      "${v}_${z_k}" => module.autoscale[v].zonal_instance_group_ids[z_k]
    }
  ]...)
```